src/UnitySampleProject/Assets/Plugins/iOS/RampChainsafeIntegration.framework/_CodeSignature/CodeResources

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Package plugins should go into the package Plugins folder, not into the UnitySampleProject.

Reply:
Also, we could create a separate package for Ramp, as it is fully on the Unity project side.
(We haven't figured out how to break projects from the core solution into separate packages yet, but the idea is to keep each feature in a separate package)

Reply:
Yeah, I'd probably make those for the entire Fiat on/off ramp (like that should be a separate package), then people can choose their own fiat provider.